### PR TITLE
Implemented sot235 (sot23-5) footprint with testing

### PR DESCRIPTION
**Pull request by anas-sarkez**
data sheet ref
https://www.bourns.com/docs/Product-Datasheets/sot23-5.pdf

### CODE REVIEW

**Comment by anas-sarkez** [comment]
@ShiboSoftwareDev  sot23-5 is ready

File: tests/sot235.test.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { test, expect } from "bun:test"
+import { convertCircuitJsonToPcbSvg } from "circuit-to-svg"
+import { fp } from "../src/footprinter"
+
+test("sot235", () => {
+  const soup = fp.string("sot235").circuitJson()
+  console.log(soup)
```
Comment (seveibar) [inline tests/sot235.test.ts:7]:
```suggestion
```

**Review by seveibar — APPROVED** [review]
the silkscreen looks weird to me on this one, we might want to check to see how jlcpcb does the silkscreen

**Comment by anas-sarkez** [comment]
> the silkscreen looks weird to me on this one, we might want to check to see how jlcpcb does the silkscreen

should be like kicad?
https://tscircuit.github.io/kicad-viewer/#Package_TO_SOT_SMD.pretty/SOT-23-5.kicad_mod

**Comment by seveibar** [comment]
@anas-sarkez yea the kicad version is fine, i think it'd be nice if it had a pin1 indicator of some kind

**Comment by anas-sarkez** [comment]
Screenshot 2024-11-08 at 9 09 01 PM
@seveibar

**Review by seveibar — APPROVED** [review]
looks great!

**Pull request merged**